### ISSUE TITLE
Try to modify the attention module in the method to your own innovative SSAM-like module, what files need to be modified?

### ISSUE THREAD

**Issue by BBABM** [Jamie-Cheung/ArtBank#7]
Is it just a matter of modifying the ldm/modules/attention.py，ldm/modules/embedding_manager.py ldm/modules/encoders/modules.py， ldm/models/diffusion/ddpm.py . Help is desperately needed, thanks!

**Comment by Jamie-Cheung** [comment]
> Is it just a matter of modifying the ldm/modules/attention.py，ldm/modules/embedding_manager.py ldm/modules/encoders/modules.py， ldm/models/diffusion/ddpm.py . Help is desperately needed, thanks!

Yes. Only modify an attention module and define a learnable parameter.

**Comment by BBABM** [comment]
> > Is it just a matter of modifying the ldm/modules/attention.py，ldm/modules/embedding_manager.py ldm/modules/encoders/modules.py， ldm/models/diffusion/ddpm.py . Help is desperately needed, thanks!
> 
> Yes. Only modify an attention module and define a learnable parameter.

thank you!
